docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -880,14 +880,28 @@ To prevent synchronization of organization roles for a specific OAuth integratio
 
 The setting `oauth_skip_org_role_update_sync` will be deprecated in favor of provider-specific settings.
 
-The following table shows the OAuth providers, the default value setting, and the skip org role sync setting.
+The table below show the OAuth provider and their setting with the default value and the skip org role sync setting.
 | OAuth Provider | `oauth_skip_org_role_sync_update` | `skip_org_role_sync` | Behavior |
 | --- | --- | --- | --- |
 | AzureAD | false | false | will sync with AzureAD roles |
 | AzureAD | true | false | skip org role sync for OAuth providers including AzureAD users |
 | AzureAD | false | true | skip org role sync for AzureAD users |
 | AzureAD | true | true | skip org role sync for AzureAD users and all other OAuth providers |
 
+### [auth.google] skip_org_role_sync
+
+Opon first login from a user we set the organization roles from the setting `AutoAssignOrgRole`. If you want to manage organizational roles, you can set the `skip_org_role_sync` option to `true`. Please note that there is also a separate setting called `oauth_skip_org_role_update_sync` which has a different scope. While `skip_org_role_sync` only applies to the specific OAuth provider, `oauth_skip_org_role_update_sync` is a generic setting that affects all configured OAuth providers.
```

Comment:
As we do not get roles from Google auth provider.

Here is the setting that we set eventually after the code goes through a user that logs in without a role assigned from the provider.
https://github.com/grafana/grafana/blob/main/pkg/services/user/userimpl/user.go#L151